debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -913,6 +914,15 @@ public static OversizeHandlingMode parse(String value, String defaultValue) {
                     + "This includes both, the size before and after database operation, "
                     + "more specifically this limits the size of fullDocument and fullDocumentBeforeChange filed of MongoDB change events.");
 
+    public static final Field CURSOR_INACTIVITY_PAUSE_MS = Field.create("cursor.inactivity.pause.ms")
```

Comment:
Do need the new config option or can we re-use `io.debezium.config.CommonConnectorConfig.getPollInterval()` is is the case with PostgreSQL?

Reply:
Probably not and we could reuse. I was just confused since this option seems to be used as an await time when the queue threshold is reached.  However the documentation for it suggests that it should be used for the use case we have here :). 

So if this overloaded use case is OK I will change things.